Technical blog related to Handling Data (BigData)#233

SamriddhiMishra opened the pull request:
#233 
<!--
Thank you for your pull request. Please review the below requirements.
-->

#### Checklist

<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] You created a branch from `dev` branch
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] Checked for all grammatical errors in the blog.
- [x] Checked for any duplicate blog entry.
- [x] You PR is raised on `dev` branch and not on`master`.

##### Description of change

<!-- In case of a bug please provide a short description of what is changed and add link of the relevant issue after this comment-->

##### Author Details for Blog Post Entry

id: Samriddhi Mishra
bio: An undergrad student pursuing BTech. Eager to learn new things.
github: SamriddhiMishra
stackoverflow: 10469993
linkedin: samriddhi-mishra
medium: samriddhi2958
twitter: samriddhi07
<!-- Please provide your details in the format specified in the [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md) -->

KarenEfereyan commented:
Another nice one @SamriddhiMishra who run the world? Data Data 🤣🤣🤣🤣

SamriddhiMishra commented:
> Another nice one @SamriddhiMishra who run the world? Data Data 🤣🤣🤣🤣

💯

puneetsingh24 reviewed (commented):
Hello @SamriddhiMishra 

We found out that a few of the content on the blog is copy-pasted from other sites. As part of our code of conduct, we encourage you to create your own content in your own language.

https://www.freecodecamp.org/news/a-quick-overview-of-the-apache-hadoop-framework/
https://www.flexiant.com/2013/07/26/what-is-distributed-storage-and-why-use-it-part-2/
https://www.smartdatacollective.com/how-much-big-data-companies-make-on-internet/


Also, please use png/jpg for the cover image, gif is not supported yet

Thanks
Can you please write these parts in your own language

SamriddhiMishra commented:
Yeah, I will try writing in my words but the facts about the generated can't be changed.

SamriddhiMishra commented:
Thanks, @puneetsingh24 for pointing out the issues. I have changed the conflicting content, but the facts can't be changed. Although I have added the link of the blog from where I collected the facts and mentioned in the blog. Please review it again.

SamriddhiMishra commented:
@puneetsingh24 Please review it. Thanks!

vyasriday commented on content/blog/handling-cheapest-fuel-data/index.md:
```diff
@@ -0,0 +1,105 @@
+author: "Samriddhi Mishra"
+coverImage: "a.png"
+tags: ["Data", "BigData", "Hadoop"]
+description: "You can have data without information, but you cannot have information without data. A large amount of data generated per data, the issues in handling. How big companies manage and analyze a huge amount of data?"
```
`A large amount of data generated per data, the issues in handling.` Is it the correct sentence @SamriddhiMishra

SamriddhiMishra replied:
@vyasriday corrected the typo. 
Thanks!

vyasriday reviewed (approved):
Thanks for updating it.

pareek-narendra commented:
@SamriddhiMishra 
Thanks for your contribution to LoginRadius Engineering Portal.

We found that the same article has already been published on medium. As per our guidelines, we do not accept articles that have already been published elsewhere. So now either you can remove your article from medium and use a canonical link from LoginRadius on medium, or we will have to remove your article form our platform.  If you need help with the canonical URL part, we can help you with that.
Anyway, we are syndicate LoginRadius blogs to all famous blog sites including medium.

https://medium.com/@samriddhi2958/handling-the-cheapest-fuel-data-6042b46f1f08

SamriddhiMishra commented:
@pareek-narendra Yeah, I would like to use the canonical link from LoginRadius and want help with that too.

pareek-narendra commented:
@SamriddhiMishra 
Please follow these steps mentioned here: https://help.medium.com/hc/en-us/articles/360033930293-Set-a-canonical-link
And this is your article link: https://www.loginradius.com/engineering/blog/handling-cheapest-fuel-data/

Please let us know once you are done or need any further help.

SamriddhiMishra commented:
@pareek-narendra I added the URL Please check
![image](https://user-images.githubusercontent.com/43378325/95321483-b8101b00-08b8-11eb-81a1-0a2316075943.png)
![image](https://user-images.githubusercontent.com/43378325/95321660-ff96a700-08b8-11eb-99bb-b230804f7ea3.png)

SamriddhiMishra commented:
@pareek-narendra Do I need to make some other changes or it is fine now?

pareek-narendra commented:
@SamriddhiMishra Thanks for making the changes, It looks good.
